lib/polyfills/fetch.js

```diff
@@ -36,22 +36,62 @@ if (typeof global.fetch === 'undefined' || !global.fetch.toString().match(/\[nat
   require('whatwg-fetch');
 
   // Export the polyfilled items that were housed in the global.self namespace.
-  module.exports = {
-    fetch: global.self.fetch,
-    Headers: global.self.Headers,
-    Request: global.self.Request,
-    Response: global.self.Response,
-  };
+  // fetch itself should be the default export, but we need to also make sure
+  // that we export the other items for destructuring in case they're needed.
+  Object.defineProperties(global.self.fetch, {
```

Comment:
I think there's some potential to simplify a bit. Just messing around here, probably not the best approach:

```javascript
function constructProperties (parent) {
  const keys = ['fetch', 'Headers', 'Request', 'Response']

  keys.forEach(key => {
    properties[key] = {
      enumerable: true,
      writeable: false,
      value: parent[key]
    }
  })
 return properties
}

// ...
Object.defineProperties(global.self.fetch, constructProperties(global.self))
// ...
Object.defineProperties(global.self.fetch, constructProperties(global))
```